docs/docs/theory_2.md

```diff
@@ -44,10 +44,94 @@ $$
+
+$$
+ \rho_\mu (t=0) = \frac{1}{2}(1 + \sigma_\mu^{\hat n}),
+$$
+
+and hence the density matrix of the full system is, (defining $d =\prod_{i \neq 0} 2I_i + 1$ as the dimension of the Hilbert space without the muon), 
+
+$$
+ \rho(t=0) = \frac{1}{2}(\mathbb{1}+\sigma_\mu^{\hat n}) \otimes \frac{1}{d}\mathbb{1}_d.
+$$
+
+Here we want to calculate the time dependence of the muon polarisation along ${\hat n}$, which is given by (notational abuse 
+means $\sigma_\mu^{\hat n}(t) = e^{iHt}(\sigma_\mu^{\hat n} \otimes \mathbb{1}_d) e^{-iHt}$)
+
+$$
+ P^{\hat n}_\mu(t) = \mathrm{Tr}[\rho(t)\sigma_\mu^{\hat n}(0)] = \mathrm{Tr}[\rho(t=0)\sigma_\mu^{\hat n}(t)].
+$$
+
+We have also switched which operator we are time-evolving on the RHS here. Now, substituting $\rho(t=0)$,  we get
+
+$$
+P^{\hat n}_\mu(t) = \mathrm{Tr}[\rho(t=0)\sigma_\mu^{\hat n}(t)]= \mathrm{Tr}\Bigg[\Big(\frac{1}{2}(\mathbb{1}+\sigma_\mu^{\hat n}) \otimes \frac{1}{d}\mathbb{1}_{d} \Big)\sigma_\mu^{\hat n}(t)\Bigg] 
+$$
+
+which can be simplified to
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\mathrm{Tr}\Bigg[\Big((\mathbb{1}+\sigma_\mu^{\hat n}) \otimes \mathbb{1}_{d} \Big)\sigma_\mu^{\hat n}(t)\Bigg].
+$$
+
+Now, if we factor out the first term in the trace, we get
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\mathrm{Tr}[(\mathbb{1} \otimes \mathbb{1}_{d}) \sigma_\mu^{\hat n}(t)]
+			+\frac{1}{2d}\mathrm{Tr}\Bigg[(\sigma_\mu^{\hat n} \otimes \mathbb{1}_{d}) \sigma_\mu^{\hat n}(t)\Bigg].
+$$
+
+Note that ass the trace of a Pauli spin matrix is **always** zero, only the second term is non-zero. Hence the muon polarisation can be written 
+as (after re-defining $\sigma_\mu^{\hat n}$ to include the kronecker product with the identity matrix of the other spins)
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\mathrm{Tr}\Bigg[\sigma_\mu^{\hat n}(0) \sigma_\mu^{\hat n}(t)\Bigg].
+$$
+
+Now replacing the trace with $\sum_\alpha \langle \alpha| ... | \alpha \rangle$, where $| \alpha \rangle$ is a complete set 
+of orthonormal eigenstates we obtain
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_\alpha \langle \alpha | \Bigg[\sigma_\mu^{\hat n}(0) \sigma_\mu^{\hat n}(t)\Bigg] | \alpha \rangle.
+$$
+
+Explicitly writing out the time dependance, we get
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha, \beta, \gamma} \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle e^{iE_\beta t} \langle \beta | \sigma_\mu^{\hat n}(0)
+|\gamma \rangle e^{-iE_\beta t} \langle \gamma | \alpha \rangle,
+$$
+
+and as $\langle \gamma | \alpha \rangle = \delta_{\gamma, \alpha}$, this simplifies to
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha, \beta} \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle e^{iE_\beta t} \langle \beta | \sigma_\mu^{\hat n}(0)
+|\alpha \rangle e^{-iE_\beta t},
+$$
+
+so that
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha, \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 e^{i(E_\beta-E_\alpha) t}.
+$$
+
+Simplifying this further we find
```

Comment:
Took me a minute to work out this last step. It's probably OK as is, maybe rather than simply `Simplifying` we could be explicit that we're expressing the exponential in terms of `Sin` and `Cos`, then using their (a)symmetry to cancel/combine terms with equal but opposite energy differences?

That's quite wordy though, what do you think?

Reply:
Yeah I think it would be worth expanding it, I would prefer being explicit so there's less chance of confusion.